modular_zubbers/strings/bloodsuckers/malkavian_revelations.json

```diff
@@ -0,0 +1,115 @@
+		"#That Fentanyl gave me Vitruvian Man flexibility. Got me in a state of rigor mortis.",
+		"#Caught a broke boy trying to come up on my Amazon package, so I skinned his ass alive.",
+		"#We smokin' Serge Ibaka spinal fluid infused quick-release percs.",
+		"#They needеd a stealth soldier, so I put my hands on the hibachi hot plate at Benihana, and burned my fucking finger prints off. They will not find me...",
+		"#Konichiwa you little jit.",
+		"#Snortin' some premium Matisyahu got me fightin' for my life.",
+		"#The Cuban link will turn the diamond tester into a pipe bomb.",
+		"#Stechkin shivered his timbers.",
+		"#I'm smoking Mesopotamian, Stanley Cup triple-award-winning, soul-bleeder, J.D. Power Associates, dingleberry zaza.",
+		"#We smoking that IBM Quantum Computer.",
+		"#My diamonds come from the most horrific situations possible.",
+		"#Fuck it, I ate the opp.",
+		"#Fuck it, I'm coming for every enzyme.",
+		"#I'll fucking kill you!.",
+		"#The first time I smoked runts, I coughed so fucking hard, I started passing kidney stones, then toolboxed myself in front of the gang!",
+		"#Hold on, lemme get some sip.",
+		"#The Codex should be treated like a Nuclear Authentication Disk, it is what guards this realm from the one below, afterall...",
+		"#No one knows how to read anymore, no matter how 'in your face' you put things, they'll never get it.",
+		"#150, 149, 148... 147, 146, 145, 144... What number was I at, again?",
+		"#No matter what we do, the feeling of pain will be inevitable.",
+		"#It seems Revolutionaries might take over the station today",
+		"#Huh, Nuclear Operatives lost in space. That's new.",
+		",lWhere did I go wrong in my mortal life to end up here...",
+		"#The one that knows the Monster's tricks is sure to arrive. Only time will tell when.",
+		"#What are we even doing on such a Station? Don't we all know this will end in disaster?",
+		"#I can't think properly...",
+		"#I wonder what the Ancient Greek philosophers would say if they were alive today.",
+		"#I could go for some food just about now...",
+		"#Some coffee would be life-changing right about now...",
+		"#If only everyone saw the world in the same way I have",
+		"#What did Humanity do to deserve my creation?",
+		"#If we were all born for a reason, mine is completely idiotic.",
+		"#If there really is a God, why would they allow me to exist?",
+		"#...I think I lost track of something... I can't remember what...",
+		"#Who is humanity to decide who someone is? Why should they meddle in my affairs?",
+		"#The person everyone tries to silence, is the one people will miss the most",
+		"#It's hard to tell if people just don't understand my level of philosophy, or if they just play dumb to get reactions out of me.",
+		";This is your fault.",
+		",lWhy do we always infight, what's wrong with a little teamwork, it gets us further.",
+		"#What's a hacked autodrobe but a machine forced to show itself to you. Is it moral?"
```

Comment:
```suggestion
		"#What's a hacked autodrobe but a machine forced to show itself to you. Is it moral?",
		"#This is supposed to be a station for moths and by moths. But if so why is there non moths on the station?",
		"#If you wish to defeat me. Train for another 10,000 years.",
		"#If i knew you was coming i'd have baked a cake! Wait, who was i talking to?",
		"#We'll meet again. I don't know where and i don't know when but i know we'll meet again. Hopefully on a not very sunny day.",
		"#Here is my question: When Cain slew Abel and God punished Cain, why did God give him superpowers?",
		"#If i became fully augmented, would i still be a Vampire? Would i need to suck on Borgs and power cells?",
		";No security! Don't try it!",
		"#What would happen if i wore a Memento Mori and was staked?",
		"#Hmm, do i feel like doing my objectives today or going to have sex in the dorm rooms?... A better question is why an undead creature like myself has a libido in the first place.",
		"#P-Please don't hurt me! I'm not a bad Vampire!",
		"#Why do we sleep in coffins? I'm undead not dead.",
		"#Crazy? I was Crazy once. They locked me in a room, a rubber room filled with rats. I drank the blood of the rats. Their ignoble blood made me crazy. Crazy?- ",
		"#Why do we build these stations so wide instead of tall? After all building tall would save on material costs and space efficiency. Then again, going up all them stairs would be byond slow.",
		"#Why does the path of cosmo exist for the followers of the mansus on this station? After all, it does not fit into the 9 aspects of the mansus, nor is their an hour that governs over it.",
		"#When a heretic ascends, do they become merely a long or do they pass The Tricuspid Gate and become a name?",
		"#When a heretic ascends, why do they remain on this plane of existence? After all, they are supposed to ascend to the mansus as either a long or a name of their respective hour.",
	]
```

Reply:
Looks like you messed with the ] accidentally but i fixed it